src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,64 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import { useSearchParams, Link } from 'react-router-dom';
+import cn from 'classnames';
+import { SexFilter } from '../types';
+import { getSearchWith } from '../utils/getSearchWith';
+
+type Props = {};
+
+const sexParams = [
+  { sortBy: SexFilter.All },
+  { sortBy: SexFilter.Male },
+  { sortBy: SexFilter.Female },
+];
```

Comment:
It's recommended to move these constants (sexParams and centuriesParams) outside of the component to avoid unnecessary re-declaration during each render. This way, you can improve the performance of your component.